cc-snapshot

```diff
@@ -98,14 +121,8 @@ fi
 export OS_REGION_NAME=$REGION
 if [ -z "$OS_REGION_NAME" ]; then unset OS_REGION_NAME; fi
 
-# if [ $KEYSTONE_VERSION = '2' ]; then
-export OS_AUTH_URL=https://chi.$SITE.chameleoncloud.org:5000/v2.0
+export OS_AUTH_URL=$(echo $CHAMELEON_OS_AUTH_URL_TMPL | sed "s/SITE/$SITE/g")
 export OS_TENANT_ID=$PROJECT_ID
-# else #keystone v3 ...migrate to this at some point
```

Comment:
Please keep the v3 code as we will likely move to it rather than keep using v2.0